test/verify/check-bots-api

```diff
@@ -84,6 +84,17 @@ class TestImageCustomize(unittest.TestCase):
         self.assertTrue(os.path.exists(img))
         self.checkBoot(img)
 
+    def testUpload(self):
+        img = os.environ["TEST_OS"]
+        with testvm.Timeout(seconds=300, error_message="Timed out waiting for image-customize"):
+            subprocess.check_call(["bots/image-customize", "--verbose",
```

Comment:
I'd feel better if you did a check_output() here and verified that grep gave you the correct result (instead of just relying on the error code).  I'd also feel better if you didn't take a copy of *my* /etc/passwd as an example :)

Reply:
The test does fail correctly without the fix. But indeed we should add a test case that `--script false` will indeed fail the image-customize command. That works now, but let's keep it that way :-)

Indeed /etc/passwd was a bit lazy, feel free to use another file.